packages/ui/src/utils/components/index.tsx

```diff
@@ -29,3 +29,10 @@ export function mergedCva<T extends object>(
         return twMergeWithConfig(classList(...classListParams));
     };
 }
+
+export function mergedCx(
```

Comment:
not sure about the name of this function 🤔

Reply:
It's the `cx` function with TW merge applied. It's following the same name conventions as `mergedCva`